File: leetcode/1347. Minimum Number of Steps to Make Two Strings Anagram/main.go
package main

func abs(a int) int {
	if a < 0 {
		return -a
	}

	return a
}

func minSteps(s string, t string) int {
	sFreq, tFreq := [26]int{}, [26]int{}

	for i := 0; i < len(s); i++ {
		sFreq[s[i]-'a']++
		tFreq[t[i]-'a']++
	}

	steps := 0
	for i := 0; i < 26; i++ {
		steps += abs(sFreq[i] - tFreq[i])
	}

	return steps / 2
}

File: leetcode/1347. Minimum Number of Steps to Make Two Strings Anagram/main.dart
class Solution {
  int minSteps(String s, String t) {
    List<int> sFreq = List<int>.filled(26, 0);
    List<int> tFreq = List<int>.filled(26, 0);

    for (int i = 0; i < s.length; i++) {
      sFreq[s.codeUnitAt(i) - 'a'.codeUnitAt(0)]++;
      tFreq[t.codeUnitAt(i) - 'a'.codeUnitAt(0)]++;
    }

    int steps = 0;
    for (int i = 0; i < 26; i++) {
      steps += (sFreq[i] - tFreq[i]).abs();
    }

    return steps ~/ 2;
  }
}
